source/adapters/cuda/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ add_ur_adapter(${TARGET_NAME}
     ${CMAKE_CURRENT_SOURCE_DIR}/../../ur/ur.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/../../ur/ur.hpp
 )
+install_ur_library(${TARGET_NAME})
```

Comment:
Why did you add this to CUDA and HIP? This wasn't needed before, right?

Reply:
No adapters were installed before. As I discussed on teams channel, they should be installed. 

Regardless of that reason, CUDA adapter is also need by my change because as I see the benchmark framework works (or at least pretends to work) on CUDA. See: https://github.com/oneapi-src/unified-runtime/blob/main/.github/workflows/benchmarks_compute.yml#L132

Reply:
I see, that makes sense.